### ISSUE TITLE
Not processing jest config file or reading wrong output file

### ISSUE THREAD

**Issue by presidenten** [yacut/tester-jest#6]
Bug report (edit: Tested tester-jest v0.4.0 and v1.0.1, tester is at v0.4.6)

Running tests in webpack project from terminal works, running tests with tester-jest fails from inside of atom.

How to verify:

- Start in terminal
```shell
git clone https://github.com/presidenten/vuex-plus-demo.git
cd vuex-plus-demo
yarn # or npm install
./node_modules/.bin/jest --color=true --config=.jestrc.json src/components/item-list/item-list-store.spec.js
# tests pass
```

- Open the folder in atom, configure config to `.jestrc.json` or add `--config=.jestrc.json` to additional args. Nothing works.

- Open `./src/components/item-list/item-list-store.spec.js` see tester-jest tests fail because of failed alias mapping that is specified in the config file.

Bonus:
- Copy the line from the output window and run it in the terminal and see the tests pass.
- Read the output file and see tests pass there as well.

**Comment by yacut** [comment]
Hallo. Thank you for the issue!
The `Jest config path` setting option was really never used. It was removed in 1.0.2 release.

But the setup via args is working.
![screenshot from 2017-05-07 21 43 36](https://cloud.githubusercontent.com/assets/4711633/25784508/783e415a-336e-11e7-95f7-45a127af3cba.png)
Unfortunately I can't reproduce the issue.
Could you please show in dev console for errors or post the command string.

**Comment by yacut** [comment]
I found the problem with `npm test --` and released it in 1.0.3.

**Comment by presidenten** [comment]
I missed that the arguments where supposed to be comma separated, I just separated with a space. I managed to get the tests green with 1.0.3, but now I get nothing except for this error message:
```
Uncaught TypeError: results.filter is not a function
~/.atom/packages/tester/lib/main.js:244
Hide Stack Trace
TypeError: results.filter is not a function
    at Tile.setResults (~/.atom/packages/tester/lib/main.js:244:37)
    at Tile.resetResults (~/.atom/packages/tester/lib/main.js:220:14)
    at ~/.atom/packages/tester/lib/main.js:89:28
    at Function.module.exports.Emitter.simpleDispatch (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/event-kit/lib/emitter.js:25:14)
    at Emitter.module.exports.Emitter.emit (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/event-kit/lib/emitter.js:129:28)
    at /Applications/Atom.app/Contents/Resources/app.asar/src/pane-container.js:348:27
    at Pane.module.exports.Pane.observeActiveItem (/Applications/Atom.app/Contents/Resources/app.asar/src/pane.js:241:7)
    at /Applications/Atom.app/Contents/Resources/app.asar/src/pane-container.js:346:42
    at Function.module.exports.Emitter.simpleDispatch (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/event-kit/lib/emitter.js:25:14)
    at Emitter.module.exports.Emitter.emit (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/event-kit/lib/emitter.js:129:28)
    at PaneContainer.module.exports.PaneContainer.setActivePane (/Applications/Atom.app/Contents/Resources/app.asar/src/pane-container.js:198:22)
    at Pane.module.exports.Pane.activate (/Applications/Atom.app/Contents/Resources/app.asar/src/pane.js:850:14)
    at Pane.module.exports.Pane.focus (/Applications/Atom.app/Contents/Resources/app.asar/src/pane.js:252:21)
    at atom-pane.<anonymous> (/Applications/Atom.app/Contents/Resources/app.asar/src/pane-element.js:50:23)
```

Any idea on what might be wrong? I've tried to reinstall tester and tester-jest and restart atom. Still the same.

**Comment by presidenten** [comment]
Ive installed 1.0.3 and fixed that I missed that the list should be comma separated.

Now the tests work. 

Thanks for the quick response!

**Comment by presidenten** [comment]
Did you by any chance forget some debugging window in 1.0.3?
I get a box in top right window that says `Jester test command: ...`. 
Clicking it expands it.

**Comment by presidenten** [comment]
Turns out something else seems to have broken down. Not sure if it is related. Any ideas?
Running tester 0.4.6 and tester-jest 1.0.3.

It happens when I try to write another spec-file in the same project.

![image](https://cloud.githubusercontent.com/assets/3883897/25785059/d054cc48-3377-11e7-8a93-5c5115058086.png)

**Comment by yacut** [comment]
This is a known issue (empty result from test provider) und it was fixed in master branch of [tester](https://github.com/yacut/tester). nfortunately I can't release it, because the package was completely rewrite with react components and dock panel which only available in atom >=1.17.

**Comment by yacut** [comment]
You can install tester from master branch (this should fix the problem), but if you use atom < 1.17 you couldn't see the console output panel.
`apm install https://github.com/yacut/tester.git`

**Comment by presidenten** [comment]
Atom released 1.17.0. Does this mean that there will be a real release soon?

**Comment by yacut** [comment]
tester > 1.0.0 should solve the problem.
